### PR TITLE
Add types for mapbox-gl-draw@1.2.1

### PR DESCRIPTION
**Pull request by tudorgergely**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/mapbox__mapbox-gl-draw/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
+// Type definitions for @mapbox/mapbox-gl-draw 1.2
+// Project: https://github.com/mapbox/mapbox-gl-draw
+// Definitions by: Tudor Gergely <https://github.com/me>
```
Comment (danger-public) [inline types/mapbox__mapbox-gl-draw/index.d.ts:3]: - :no_entry_sign: This line should have your github username in it, not /me

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mapbox__mapbox-gl-draw ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mapbox__mapbox-gl-draw@latest/))
was missing the following properties:
1. modes
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against acdddb8332ab1a574820c87c8f345f56be2e9bbc
</p>

**Comment by brookjordan** [comment]
Oh wow, great timing. We’ve just started using draw and were looking for this!

**Review by brookjordan — APPROVED** [review]
Oh wow, great timing. We’ve just started using draw and were looking for this.
I think this all makes sense!

**Comment by tudorgergely** [comment]
Ready to merge

**Review by orta — APPROVED** [review]
Looks great to me

**Pull request merged**